### ISSUE TITLE
When running clin.iobio using a configuration file it freezes on Review variants

### ISSUE THREAD

**Issue by JasperO98** [iobio/clin.iobio#487]
When running clin.iobio using a configuration file, [Case3_configuration.csv](https://github.com/iobio/clin.iobio/files/10003072/Case3_configuration.csv), it correctly loads the files for the quality control step and phenotype selection, but it is stuck at the Review variants step. It shows the box with "Loading files", but nothing happens.

![image](https://user-images.githubusercontent.com/38041391/201664649-994f2c0e-6d4b-4733-9120-718e5e83a7bb.png)

**Comment by tonydisera** [comment]
Hi Jasper -

It looks like the vcf file only contains variants for chr11. You should be seeing warnings like this: 
<img width="756" alt="Screenshot 2022-11-29 at 7 26 49 PM" src="https://user-images.githubusercontent.com/8420103/204692817-ee97f6b4-86b0-4b76-9294-c9541527d45b.png">

**Comment by JasperO98** [comment]
Yes I can access the Review Variants tab now and I can see the warning messages! Thank you.

![image](https://user-images.githubusercontent.com/38041391/204757090-876e97e3-ad19-4597-bfd1-c998e08a2d26.png)
